src/Core/Console/Command.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Console;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+
+/**
+ * Symfony Command class for PrestaShop allowed to rely on legacy classes
+ *
+ * @author Mickaël Andrieu <mickael.andrieu@prestashop.com>
+ */
+abstract class Command extends ContainerAwareCommand
```

Comment:
Could we improve the naming ? `Command` is a bit shy to express the fact that this is a "Symfony Command class for PrestaShop allowed to rely on legacy classes"

Reply:
I formerly named it "PrestaShopCommand", but as this class is part of PrestaShop only I don't see the value to prefix the name of my class with PrestaShop.

I'm open to suggestions, what do you suggest?